index.html

```diff
@@ -1096,8 +1096,14 @@ <h2><dfn data-lt="implicitly release the pointer capture|implicitly releasing po
                 if the pointer supports hover, user agent MUST also send corresponding boundary events necessary
                 to reflect the current position of the pointer with no capture.</p>
             <p>When the <a>pointer capture target override</a> is no longer [=connected=] [[DOM]],
-                the <a>pending pointer capture target override</a> and <a>pointer capture target override</a> nodes SHOULD be cleared
-                and also a PointerEvent named {{GlobalEventHandlers/lostpointercapture}} corresponding to the captured pointer SHOULD be fired at the document.</p>
+                the <a>pointer capture target override</a> SHOULD be set to the document.</p>
+            <p>When the <a>pending pointer capture target override</a> is no longer [=connected=] [[DOM]],
+                the <a>pending pointer capture target override</a> node SHOULD be cleared.</p>
+            <div class="note">
+                The previous two paragraphs result in a {{GlobalEventHandlers/lostpointercapture}} event
```

Comment:
I think this note clarifies the outcome of _only_ the first paragraph modified here.  Regardless of what happens to the "pending state", the next invocation of "Process pending pointer capture" will fire a `lostpointercapture` to document.

I suggest making this a normative text at the end of the first para above.

Reply:
This note clarifies the outcome of *both* paragraphs. If we only had the first paragraph, the pending pointer capture target override would still be the removed element, so we would attempt to re-apply pointer capture to the removed element.

> I suggest making this a normative text at the end of the first para above.

I don't think this is standard practice, this note is clarification as to what will happen as a result of the normative requirements. We don't normally clarify with normative statements as it could result in inconsistencies between the normative requirement and the clarifying paragraph and the paragraph.

In particular, this clarifying paragraph only describes one very specific case - but not the plethora of cases that can happen, e.g. if you request capture again on the node after re-adding the node, you'll still get a lostpointercapture.

Reply:
Sounds good.